src/components/map/layers/NodeLayer.tsx

```diff
@@ -71,7 +89,11 @@ export default class NodeLayer extends Component<MarkerLayerProps> {
             break;
         }
         default: {
-            throw new Error(`NodeType not supported: ${node.type}!`);
+            this.props.notificationStore!.addNotification({
+                message: `Solmu id: '${node.id}', tyyppi '${node.type}' on virheellinen`,
```

Comment:
How about format "Solmun 123123 (id) tyyppi puuttuu."

And is this really something that should be notified? Ask customer perhaps.

Reply:
Other solutions: a yellow/red exclamation mark next to that [ i ] button in sidebar (routeShow). Clicking that [ ! ] button you could see a small window of all invalid properties or smth...

Reply:
We can't really say that it is missing, since we don't have that information. The only thing we can say that it is invalid, since it cannot be mapped to the mappings we have in the UI.
And I think it is pretty clear that this should be visible to the users, finding errors such as these is one important improvement over the old program.